NAMESPACE

```diff
@@ -21,6 +21,7 @@ export(dumpCounts.scalpel)
 export(filterForDeaminase)
 export(filterForSignature)
 export(getAltAllele)
+export(getRefAllele)
```

Comment:
It looks like there is still an export of the removed getRefAllele function, may be worth rerunning Roxygen